tests/vagrant/test.yml

```diff
@@ -13,7 +13,7 @@
         - build
 
   roles:
-    - name: ansible-city.nginx
+    - role: ansible-city.nginx
```

Comment:
I have swapped `ansible-city` with `sansible` and squashed all commits into one. I believe it should be ready for merging.